vividus-tests/src/main/resources/story/integration/HttpCookieSteps.story

```diff
@@ -7,3 +7,9 @@ Scenario: Verify step: "When I save value of HTTP cookie with name $cookieName t
 When I send HTTP GET to the relative URL '/cookies/set?cookieName=cookieValue'
 When I save value of HTTP cookie with name `cookieName` to SCENARIO variable `value`
 Then `${value}` is equal to `cookieValue`
+
+Scenario: Verify step: "I change value of all HTTP cookies with name `$cookieName` to `$newCookieValue`"
+When I send HTTP GET to the relative URL '/cookies/set?cookieName=cookieValue'
```

Comment:
use another `cookieName` to make sure the previous scenario doesn't affect this one